_posts/2019-06-11-sgid-then-and-now.md

```diff
@@ -0,0 +1,128 @@
+  - Elevation and Terrain - (20,400)
+  - Aerial Photography - (20,100)
+  - Transportation - (18,200)
+  - USGS Scanned Topo Maps - (15,900)
+  - Boundaries - Landownership - (15,600)
+  - Boundaries - City, County, and State - (15,300)
+  - Oil and Gas - (8,000)
+  - Water - (7,600)
+  - Basemap and Imagery - (5,300)
+
+- **[ArcGIS Online](https://utah.maps.arcgis.com)** - The most popular content from June 2013 to May 2019, rounded to the nearest one-thousand
+  - SBI Operations (which includes: Law Enforcement Boundaries, City Courts, County Courts, State District Courts, UHP Dispatch Zones) - (67,000 views)
+  -  Roads (minor and major roads combined stats) - (64,000)
+  -  County Boundaries - (47,000)
+  -  Municipal Boundaries - (35,000)
+  -  Parcels - (28,000)
+  -  Lakes - (26,000)
+  -  Address Points - (17,000)
+  -  Zip Codes - (17,000)
+
+- **SQL ArcSDE Database** - Desktop-user Views, from the past 12 months (approximately), rounded to the nearest one-hundred
+  - Parcels, all counties combined - (14,000 table-locks)
+  - Counties - (8,000)
+  - Municipalities - (6,100)
+  - Landownership - (4,900)
+  - Roads - (4,700)
+  - Sections - (4,000)
+  - Townships - (3,600)
+  - Quarter-Quarter Sections - (1,500)
+  - Zip Codes - (2,100)
+  - Address Points - (2,100)
+
+- **[OpenData website](https://opendata.gis.utah.gov/)**
+  - Currently, averaging 48 item views a day (17,413 views over the past year)
+
+### `Where is the SGID Going?`
+{: .text-left}
+
+One of the challenges of working in technology is that over time file formats, and the way we access data, change.  This is similarly true for the SGID.  As a result, AGRC engages in regular discussions on where the SGID should be going.  
+
+Currently, we are working toward [trimming down the SQL ArcSDE enterprise database]({{site.baseurl}}{% post_url 2018-09-10-the-incredible-shrinking-sgid %}), which is available to those on the state network.  Moving forward, we will maintain only ‘live’ datasets in the SDE.  These would be datasets that receive regular modifications, such as address points, parcels, municipal boundaries, roads, etc. The idea is to use this database as our internal, editing, and staging database.
+
+All data layers being maintained in the SDE will also be available in [ArcGIS Online (AGOL)](https://utah.maps.arcgis.com) and [AGRC’s open data portal](https://opendata.gis.utah.gov/).  Here users can consume the web service or download the data.  Obsolete data will be written to object storage for retention and available on request.  
+
+In the near future, AGRC will also offer a public-facing, PostgreSQL cloud database where users can make direct connections from their desktop client.  This will bring new functionality as well as an additional access point for our end users.  We are excited about this offering so keep an eye on our [Twitter](https://twitter.com/maputah) or [newsletter](https://gis.utah.gov/news/#mc_embed_signup) for upcoming details.  
```

Comment:
Are we getting ahead of ourselves here? Are we planning to test this first before committing to it or are we all in?

Reply:
we could say we're 'testing the option'.  though, i have an arcgis desktop connection to the postgres database that keaton built and a few of us have tested it. out a bit  it seems pretty solid.  I guess we just need the data quality checks and the update cycle ironed out.   what do you think, 'testing the option'?

Reply:
that seems like safe wording